docs/snippets/svelte/msw-addon-configure-handlers-graphql.js.mdx

```diff
@@ -8,6 +8,9 @@ import MockApolloWrapperClient from './MockApolloWrapperClient.svelte';
 
 export default {
   component: DocumentScreen,
+  render: () => ({
```

Comment:
is this needed at all? seems like it would be equivalent to the default render function